### PR TITLE
fix: integrate timeline and graph tabs wrappers under one component

### PR DESCRIPTION
**Pull request by olga-union**
Signed-off-by: Olga Nad <olga@union.ai>

# TL;DR
Execution page refactor: created `ExecutionTab` component as a wrapper for every tab on the node executions page, so all of them consume the same data, and became more unified in nature. Currently, integration takes on the Graph and Timeline tabs, while bringing Executions Table to the same structure is in the future plans.

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [x] Chore
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Demo
_Map task_ 
https://user-images.githubusercontent.com/101579322/187541242-1a4063b5-cb50-459c-aa86-83b2bb139c51.mov
_Simple wf_ 
https://user-images.githubusercontent.com/101579322/187541325-0952909a-db5b-41e2-8814-dad8dfa82946.mov
_Sub-workflows_ 
https://user-images.githubusercontent.com/101579322/187541377-33f937d4-5ffa-463c-88f9-0ca33ccf4d41.mov

## Tracking Issue
_NA_

## Follow-up issue
_NA_

### CODE REVIEW

File: packages/zapp/console/src/components/Executions/ExecutionDetails/Timeline/taskNames.tsx [hunk 1]
```diff
@@ -55,7 +54,6 @@ interface TaskNamesProps {
 }
 
 export const TaskNames = React.forwardRef<HTMLDivElement, TaskNamesProps>((props, ref) => {
```
Comment (olga-union) [inline packages/zapp/console/src/components/Executions/ExecutionDetails/Timeline/taskNames.tsx:56]: renamed the file to start with capital letter, as it's component

**Review by jsonporter — APPROVED** [review]
lgtm
